Library/Homebrew/extend/os/mac/keg.rb

```diff
@@ -0,0 +1,4 @@
+class Keg
+  GENERIC_KEG_LINK_DIRECTORIES = remove_const :KEG_LINK_DIRECTORIES
```

Comment:
Changing constant definitions feels weird to me but I don't have a better suggestion (maybe @reitermarkus does)? 👍 to the PR as a whole, thanks @sjackman ❤️

Reply:
In the past we've simply changed the constant to a function, `keg_link_directories`. I have a mild preference for the constant, but I'm fine with either.